### ISSUE TITLE
Accessibility Bugs

### ISSUE THREAD

**Issue by santwuma** [NCIOCPL/gatsby-pcp#140]
- Links at bottom don’t actually read the text. Our “hidden” class seems to hide from the screen reader.
- Main card panels are duplicative. Images should be alt text “” since it’s the same as the title.
- Side nav drops you into a “group” when you visit it. Not sure if this is a problem.
- Home logo should not be in a list. It reads funny.
- Top area should calls itself a “banner”. Is there a better role we can assign to it for navigation purposes?
- aria-hidden is false around navigation area makes it impossible to navigate through it.
- Reports page should have h4 tags for “Associated meetings”
